Feature/0.2fixes

Dumpy250 opened the pull request:
- This includes several fixes on Azadi, Lumberman, Valley, Shield
- ~~Added Bandar and Alborz save files~~
fixes: https://github.com/BF3RM/RealityMod/issues/1840

Dumpy250 committed:
feature(0.2fixes): Chris's new fixes

Karkand, Shield, and Death Valley

Dumpy250 committed:
feature(0.2fixes): Valley fixes

More valley fixes, removed dead trees under bridges

Dumpy250 committed:
feature(0.2fixes): Several Fixes on Azadi

-Fixed several slightly floating objects
-Fixed a wall players could walk through

Dumpy250 committed:
feature(0.2fixes): Revert of Karkand

- Reverted Karkand

Dumpy250 committed:
feature(0.2fixes): Added Alborz save file

- Added Alborz save file
- Added Alborz to gamemode json

Dumpy250 committed:
feature(0.2fixes) Implemented Bandar

- Added Sassy's Save file
- Added to gamemode_map

Dumpy250 committed:
feature(0.2fixes): lumberman fixes

- Fixed floating rocks

FoolHen commented:
As mentioned, the new maps should be on their own branch (i would even do one per map), as they are not finished

Dumpy250 committed:
feat(0.2fixes): refactored save files

- refactored save files
- removed Alborz and Bandar (will move to another branch)
